dnsapi/dns_variomedia.sh

```diff
@@ -93,11 +89,12 @@ dns_variomedia_rm() {
 # _sub_domain=_acme-challenge.www
 # _domain=domain.com
 _get_root() {
-  fulldomain=$1
-  i=1
+  domain=$1
+  i=2
```

Comment:
the value of "i" should be begging from `1`, not `2`.

In dns alias mode,  the fulldomain can be no  `_acme-challenge`  prefix

Reply:
it's been tested:
- successfully with i=2
- unsuccessfully with i=1